### PR TITLE
Libva

### PR DESCRIPTION
**Pull request by okias**

### CODE REVIEW

**Comment by okias** [comment]
@mattst88 if possible, after maintainer change, reassign to x11 team :)

**Comment by aballier** [comment]
would be great if you could use this PR to bump x11-libs/libva-intel-driver too

**Comment by okias** [comment]
@aballier I must missed that one, does it look OK now?

Thanks

**Comment by candrews** [comment]
@aballier if you're busy and want me to merge this, I'm happy to do it for you. Let me know!

File: x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	SRC_URI=""
+else
+	SRC_URI="https://github.com/intel/intel-vaapi-driver/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/intel-vaapi-driver-${PV}"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+if [ "${PV%9999}" = "${PV}" ] ; then
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
```
Comment (a17r) [inline x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: merge this with the above if-else

File: x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+else
+	KEYWORDS=""
```
Comment (a17r) [inline x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: drop this line entirely

File: x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	EGIT_BRANCH=master
+	EGIT_REPO_URI="https://github.com/intel/intel-vaapi-driver"
+fi
+
+AUTOTOOLS_AUTORECONF="yes"
+inherit autotools-multilib ${SCM}
+
+DESCRIPTION="HW video decode support for Intel integrated graphics"
+HOMEPAGE="https://github.com/intel/intel-vaapi-driver"
+if [ "${PV%9999}" != "${PV}" ] ; then # Live ebuild
```
Comment (a17r) [inline x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: double brackets please

File: x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+SCM=""
+if [ "${PV%9999}" != "${PV}" ] ; then # Live ebuild
```
Comment (a17r) [inline x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: double brackets please

File: x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+if [ "${PV%9999}" = "${PV}" ] ; then
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+else
+	KEYWORDS=""
+fi
+IUSE="wayland X"
+
+RDEPEND=">=x11-libs/libva-2.0.0:=[X?,wayland?,drm,${MULTILIB_USEDEP}]
+	>=x11-libs/libdrm-2.4.52[video_cards_intel,${MULTILIB_USEDEP}]
+	wayland? ( >=media-libs/mesa-9.1.6[egl,${MULTILIB_USEDEP}] >=dev-libs/wayland-1.11[${MULTILIB_USEDEP}] )"
```
Comment (a17r) [inline x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: one line per dependency.

File: x11-libs/libva/libva-2.4.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+SCM=""
+if [ "${PV%9999}" != "${PV}" ] ; then # Live ebuild
```
Comment (a17r) [inline x11-libs/libva/libva-2.4.0.ebuild]: double brackets please

File: x11-libs/libva/libva-2.4.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+	EGIT_BRANCH=master
+	EGIT_REPO_URI="https://github.com/intel/libva"
+fi
+
+AUTOTOOLS_AUTORECONF="yes"
+inherit autotools-multilib ${SCM} multilib versionator
+
+DESCRIPTION="Video Acceleration (VA) API for Linux"
+HOMEPAGE="https://01.org/linuxmedia/vaapi"
+if [ "${PV%9999}" != "${PV}" ] ; then # Live ebuild
```
Comment (a17r) [inline x11-libs/libva/libva-2.4.0.ebuild]: double brackets please

File: x11-libs/libva/libva-2.4.0.ebuild [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+	SRC_URI=""
+else
+	SRC_URI="https://github.com/intel/libva/archive/${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+LICENSE="MIT"
+SLOT="0/$(get_version_component_range 1)"
+if [ "${PV%9999}" = "${PV}" ] ; then
+	KEYWORDS="~amd64 ~arm64 ~x86 ~amd64-linux ~x86-linux"
```
Comment (a17r) [inline x11-libs/libva/libva-2.4.0.ebuild]: merge this with above if-else.

File: x11-libs/libva/libva-2.4.0.ebuild [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+else
+	KEYWORDS=""
```
Comment (a17r) [inline x11-libs/libva/libva-2.4.0.ebuild]: drop this line entirely

**Comment by a17r** [comment]
There were no upstream changes to the package that would make changes to the ebuild necessary?

**Comment by okias** [comment]
changes incorporated

File: x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="wayland X"
+
+RDEPEND=">=x11-libs/libva-2.0.0:=[X?,wayland?,drm,${MULTILIB_USEDEP}]
+	>=x11-libs/libdrm-2.4.52[video_cards_intel,${MULTILIB_USEDEP}]
+	wayland? ( >=media-libs/mesa-9.1.6[egl,${MULTILIB_USEDEP}]
+				>=dev-libs/wayland-1.11[${MULTILIB_USEDEP}]
```
Comment (a17r) [inline x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: that's two tabs too many ;)

What about my question about upstream changes - did you check changelog and buildsystem?

Comment (okias) [reply x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: tabs depends on editor.
upstream changes are minimal and of course it works and builds

Comment (a17r) [reply x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: > tabs depends on editor.
line 32 has one tab, line 33 has four tabs - that's nothing to do with your editor.

Comment (okias) [reply x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: so let's be it space

Comment (a17r) [reply x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: no space either! 'two tabs too many' plus 'don't rely on visual length of your editor tab setting' equals *tab indentation only, one level after another, don't try to align more fine-grained*.

if you want to align those two dependencies, just make a linebreak after `wayland? (` and put them on entirely separate lines.

Comment (okias) [reply x11-libs/libva-intel-driver/libva-intel-driver-2.3.0.ebuild]: are u happy now?! :D fixed

**Comment by okias** [comment]
I'll try do some time and port it, but not giving promise :)

**Pull request closed without merging**